doc/getting-started.rst

```diff
@@ -0,0 +1,596 @@
+
+- We used :doc:`variable substitutions
+  <topics/variables-extending-and-substitutions>`:
+
+  ``${buildout:directory}``
+      Expands to the full path of the buildout directory.
+
+  ``${buildout:bin-directory}``
+      Expands to the full path of the buildout's ``bin`` directory.
+
+  Variable substitution provides a way to access Buildout settings and
+  share information between parts and avoid repetition.
+
+  See the :doc:`reference <reference>` to see what buildout settings
+  are available.
+
+- We spread the program over multiple lines.  A configuration value
+  can be spread over multiple lines as long as the continuation lines
+  begin with whitespace.
+
+  The interpretation of a value is up to the recipe that uses it. The
+  ``zc.zdaemonrecipe`` recipe combines the program value into a single
+  line.
+
+If we run Buildout::
+
+  buildout
+
+- The ``zc.zdaemonrecipe`` recipe will be downloaded and installed in
+  the eggs directory.
+
+- A ``server`` script is added to the ``bin`` directory.  This script
+  is generated by the recipe.  It can be run like::
+
+    bin/server start
+
+  to start a server and::
+
+    bin/server stop
+
+  to stop it.  The script references a zdaemon configuration file
+  generated by the recipe in ``parts/server/zdaemon.conf``.
+
+- A zdaemon configuration script is generated in
+  ``parts/server/zdaemon.conf`` that looks something like::
+
+    <runner>
+      daemon on
+      directory /Users/jim/t/0214/parts/server
+      program /Users/jim/t/0214/bin/runzeo -f /Users/jim/t/0214/data.fs -a 127.0.0.1:8200
+      socket-name /Users/jim/t/0214/parts/server/zdaemon.sock
+      transcript /Users/jim/t/0214/parts/server/transcript.log
+    </runner>
+
+    <eventlog>
+      <logfile>
+        path /Users/jim/t/0214/parts/server/transcript.log
+      </logfile>
+    </eventlog>
+
+  The **details aren't important**, other than the fact that the
+  configuration file reflects part options and the actual buildout
+  location.
+
+Version control
+===============
+
+In this example, the only file that needs to be checked into version
+control is the configuration file, ``buildout.cfg``.  Everything else
+is generated.  Someone else could check out the project, and get the
+same result [#if-same-environment]_.
+
+More than just a package installer
+==================================
+
+The example shown above illustrates how Buildout is more than just a
+package installer, like ``pip``. Using Buildout recipes, we can
+install custom scripts and configuration files, and much more. For
+example, we could use `configure and make
+<https://pypi.python.org/pypi/zc.recipe.cmmi>`_ to install non-Python
+software from source, we could run JavaScript builders, or do anything
+else that can be automated with Python.
+
+Buildout is a simple automation framework.  There are hundreds of
+recipes to choose from and :doc:`writing new recipes is easy
+<topics/writing-recipes>`.
```

Comment:
It would be good to let the user know how they can find those hundreds of recipes to choose from.
Do we have we list?  Maybe a PyPI search page?  Is there a Trove classifier for Buildout recipes?
(A short curated list of the most common/most useful/best recipes in a sibling document could be handy as well.)